src/formatter/formatProp.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+
+import spacer from './spacer';
+import formatPropValue from './formatPropValue';
+import type { Options } from './../options';
+
+export default (
+  propName: string,
+  hasValue: boolean,
+  value: any,
+  hasDefaultValue: boolean,
+  defaultValue: any,
+  inline: boolean,
+  lvl: number,
```

Comment:
The `lvl` is the current formatting depth. Could be interpreted as the current indentation level. I do not change this parameter behavior. It could be rename to something more obvious